network.tf

```diff
@@ -0,0 +1,35 @@
+resource "azurerm_resource_group" "ts4" {
+  name     = "${var.prefix}-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_virtual_network" "main" {
+  name                = "${var.prefix}-network"
+  address_space       = ["10.0.0.0/16"]
+  location            = azurerm_resource_group.ts4.location
+  resource_group_name = azurerm_resource_group.ts4.name
+}
+
+resource "azurerm_subnet" "internal" {
+  name                 = "internal"
+  resource_group_name  = azurerm_resource_group.ts4.name
+  virtual_network_name = azurerm_virtual_network.main.name
+  address_prefixes     = ["10.0.2.0/24"]
```

Comment:
It would be better to move this value to variables

Reply:
fix (add to variavles)